subprojects/instant-execution/src/integTest/groovy/org/gradle/instantexecution/AbstractInstantExecutionIntegrationTest.groovy

```diff
@@ -27,4 +28,22 @@ class AbstractInstantExecutionIntegrationTest extends AbstractIntegrationSpec {
 
     static final String INSTANT_EXECUTION_PROPERTY = "-Dorg.gradle.unsafe.instant-execution"
 
+    protected BuildOperationsFixture newBuildOperationsFixture() {
```

Comment:
❓ How about introducing an `InstantExecutionBuildOperationsFixture` class with those `assert*` methods turned into predicates (`operations.(stateLoaded|stateStored|none)`)?

Reply:
Thanks for the suggestion, did just that with different names